package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "oleg-starter-pack",
-    "version": "2.3.3",
+    "version": "2.3.4",
```

Comment:
@marymooney please can you revert this then use the shiny new release workflow

Reply:
Ahh okey dokes. I was wondering whether we still needed this with the new workflow :) Will do